src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -43,6 +43,10 @@
             ENABLED, DISABLED
         }
 
+        public enum AutoClimbState {
+            GROUND_MID, MID_HIGH, HIGH_TRAV
```

Comment:
State gets adjusted at the beginning of a command so robot state may drift from programmatic state.
For the next iteration of improvements, consider whether "in progress" or "triggered" states in between             GROUND_MID, MID_HIGH, HIGH_TRAV that may help recover a climb step that needs to be paused (ie get bumped by an alliance partner).